packages/core/src/test/shared/telemetry/util.test.ts

```diff
@@ -107,6 +110,19 @@ describe('TelemetryConfig', function () {
     })
 })
 
+describe('getSessionId', function () {
+    it('returns a random UUIDv4', function () {
+        const result = getSessionId()
+
+        assert.deepStrictEqual(isUUID(result), true)
+        assert.deepStrictEqual(getSessionId(), result, 'Subsequent call did not return the same UUID')
+    })
+
+    it('nonce is the same as always', function () {
+        assert.deepStrictEqual(sessionIdNonce, '44cfdb20-b30b-4585-a66c-9f48f24f99b5')
```

Comment:
do we need to test that a constant is constant?

Reply:
Any idea how this is possible? I couldn't find one